submissions/zalevskyi/html-css-popup/input.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Pop-up CSS only</title>
+        <link rel="stylesheet" href="./css/popup-general.css">
+        <link rel="stylesheet" href="./css/popup-input.css">
+    </head>
+    <body>
+        <nav>
+            <div id="menu_main">
+                <a href="#" class="nav_text_link">About Us</a>
+                <input type="checkbox" id="menu_recycle_checkbox"/>
+                <label for="menu_recycle_checkbox" class="nav_text_link"><img src="img/041-recycle.svg"></label>
+                <a href="#" class="nav_text_link">Contacts</a>
```

Comment:
Wrap everything inside your block #menu_main (don't forget to replace all your id on class) in `ul` | `ol` , links wrap in `li` items